### DISCUSSION TITLE
transmission-daemon 4.0.0-dev (671f835228) CMake Error at /usr/share/cmake-3.25/Modules/ExternalProject.cmake:3115 (message):

### DISCUSSION THREAD

**Discussion by sSiDs** [transmission/transmission#5439] [Support]
### What is the issue?

get error while installing 4.0.3
`-- Found CURL: /usr/lib/x86_64-linux-gnu/libcurl.so (found suitable version "7.88.1", minimum required is "7.28.0")  
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found suitable version "3.0.8", minimum required is "0.9.7")  
-- Checking for modules 'gtkmm-4.0>=3.24.0;glibmm-2.68>=2.60.0;giomm-2.68>=2.26.0'
--   Package 'gtkmm-4.0', required by 'virtual:world', not found
--   Package 'glibmm-2.68', required by 'virtual:world', not found
--   Package 'giomm-2.68', required by 'virtual:world', not found
-- Checking for modules 'gtkmm-3.0>=3.24.0;glibmm-2.4>=2.60.0;giomm-2.4>=2.26.0'
--   Package 'gtkmm-3.0', required by 'virtual:world', not found
--   Package 'glibmm-2.4', required by 'virtual:world', not found
--   Package 'giomm-2.4', required by 'virtual:world', not found
-- Found DEFLATE: /usr/include (found suitable version "1.14", minimum required is "1.7") 
-- Could NOT find EVENT2 (missing: EVENT2_LIBRARY EVENT2_INCLUDE_DIR) (Required is at least version "2.1.0")
CMake Error at /usr/share/cmake-3.25/Modules/ExternalProject.cmake:3115 (message):
  No download info given for 'event' and its source directory:

   /home/sid/transmission-4.0.3/third-party/libevent

  is not an existing non-empty directory.  Please specify one of:

   * SOURCE_DIR with an existing non-empty directory
   * DOWNLOAD_COMMAND
   * URL
   * GIT_REPOSITORY
   * SVN_REPOSITORY
   * HG_REPOSITORY
   * CVS_REPOSITORY and CVS_MODULE
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/ExternalProject.cmake:4185 (_ep_add_download_command)
  cmake/TrMacros.cmake:170 (ExternalProject_Add)
  CMakeLists.txt:467 (tr_add_external_auto_library)

-- Configuring incomplete, errors occurred!
See also "/home/sid/transmission-4.0.3/build/CMakeFiles/CMakeOutput.log".
sid@kino:~/transmission-4.0.3/build$ less home/sid/transmission-4.0.3/build/CMakeFiles/CMakeOutput.log
home/sid/transmission-4.0.3/build/CMakeFiles/CMakeOutput.log: No such file or directory
sid@kino:~/transmission-4.0.3/build$ `

### Which application of Transmission?

None

### Which version of Transmission?

4.0.0

**Comment by ckerr** [comment]
> -- Package 'gtkmm-4.0', required by 'virtual:world', not found
> -- Package 'glibmm-2.68', required by 'virtual:world', not found
> -- Package 'giomm-2.68', required by 'virtual:world', not found
> -- Checking for modules 'gtkmm-3.0>=3.24.0;glibmm-2.4>=2.60.0;giomm-2.4>=2.26.0'
> -- Package 'gtkmm-3.0', required by 'virtual:world', not found
> -- Package 'glibmm-2.4', required by 'virtual:world', not found
> -- Package 'giomm-2.4', required by 'virtual:world', not found

Hi @sSiDs, thanks for reaching out!

I'm going to convert this to a discussion topic since it doesn't appear to be a Transmission bug. Looks like you don't have the prerequisites gtkmm-dev, glibmm-dev, and giomm-dev installed?

**Comment by barracuda156** [comment]
This is precisely how it fails in Macports too, on newer macOS.
